### PR TITLE
ZW189 Range Extender 7 and ZW175 Smart Switch 7 support

### PR DESCRIPTION
**Pull request by Aeotec-ccheng**
Integration of ZW175 Smart Switch 7
- Support for ON/OFF control.
- Support for Watt and kWh reports.
- Support for all parameter settings.

### CODE REVIEW

**Comment by Aeotec-ccheng** [comment]
Integration of ZW189 Range Extender 7
- Basic Repeater Integration
- No Parameter settings available

**Comment by KyleAthom** [comment]
Hi @Aeotec-ccheng ,

Is this PR ready for merging?

**Comment by Aeotec-ccheng** [comment]
Hey @KyleAthom,

Yup, all ready to be merged!

File: drivers/ZW175/device.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const ZwaveDevice = require('homey-meshdriver').ZwaveDevice;
+
+class AeotecSmartSwitchSevenDevice extends ZwaveDevice {
+	
+	onMeshInit() {
+		this.registerCapability('onoff', 'SWITCH_BINARY');
+		this.registerCapability('measure_power', 'METER');
+		this.registerCapability('meter_power', 'METER');
+       
+		this.registerSetting('80', value => new Buffer([(value) ? 2 : 0]));
```
Comment (KyleAthom) [inline drivers/ZW175/device.js:12]: new Buffer will be deprecated in the future. It's advised to use Buffer.from() instead.

File: drivers/ZW175/driver.settings.compose.json [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+			{
+				"id": "0",
+				"label": {
+					"en": "Disable, no reaction."
+				}
+			},
+			{
+				"id": "1",
+				"label": {
+					"en": "Switch turns ON"
```
Comment (KyleAthom) [inline drivers/ZW175/driver.settings.compose.json:46]: On and off may be spelled without caps to keep all text on Homey consistent.

File: drivers/ZW175/driver.settings.compose.json [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+		"value": 2
+	},
+	{
+		"id": "19",
+		"type": "number",
+		"label": {
+			"en": "Minimum wattage change (%) needed before an update report is send"
+		},
+		"hint": {
+			"en": "This parameter ({{zwaveParameterIndex}}) is a SET only command, setting this parameter will cause the LED to blink in for X amount of seconds before it stops on its own."
```
Comment (KyleAthom) [inline drivers/ZW175/driver.settings.compose.json:130]: ".. for the set amount of time." looks neater then "..X amount"

File: drivers/ZW175/device.js (hunk 1 above)
Comment (Aeotec-ccheng) [reply drivers/ZW175/device.js:12]: So in this case, would the correct change be: 

Buffer.from([(value) ? 2 : 0])

If so, i'll include this change in the next pull request.

File: drivers/ZW175/driver.settings.compose.json (hunk 2 above)
Comment (Aeotec-ccheng) [reply drivers/ZW175/driver.settings.compose.json:46]: Got it, this will change in the next pull request.

**Pull request merged**